### PR TITLE
Intial install

### PR DESCRIPTION
**Pull request by heintayzar-hm**
Be sure to reference the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) from the sneak peek if you have any questions about table or column names or data relationships.

- Create a new app. Give it a meaningful name.
- Initialize your project with Git.
- Make sure that your project has Postgres database set up.
- Build your project schema.
    - Create and run the necessary migration files.
    - Table and column names should match the ERD diagram.
        - note: photo for users table should be a link to an image
    - Foreign keys should be included.
    - All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/migrate/20230223075432_add_column_to_users.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class AddColumnToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :name, :string
+    add_column :users, :photo, :string
+    add_column :users, :bio, :string
```
Comment (Ismailco) [inline db/migrate/20230223075432_add_column_to_users.rb]: - [ ] Great job on creating the tables however the `bio` should be a `text`, not `string`, kindly make sure to change the bio type to text so your migrations reflect the `ERD`

File: db/migrate/20230223075620_add_column_to_posts.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class AddColumnToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_column :posts, :title, :string
+    add_column :posts, :text, :string
```
Comment (Ismailco) [inline db/migrate/20230223075620_add_column_to_posts.rb]: - [ ] Same here the `text` should be a `text`, not a `string`, kindly make sure to change the text type to `text` so your migrations reflect the `ERD`

File: db/migrate/20230223075702_add_column_to_comments.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddColumnToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_column :comments, :text, :string
```
Comment (Ismailco) [inline db/migrate/20230223075702_add_column_to_comments.rb]: - [ ] Same here the `text` should be a `text`, not a `string`, kindly make sure to change the text type to `text` so your migrations reflect the `ERD`

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @heintayzar-hm, 

Good job so far 👍🏼 
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights

✔️ Used Gitflow
✔️  No linter errors
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] [OPTIONAL] in my opinion it would be better if you used fewer files for your migration tables, like creating the columns directly when you're creating the table and grouping the foreign keys of the same table in the same file instead of creating new files 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @heintayzar-hm      , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- No linter errors




Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**